copier.yml

```diff
@@ -25,6 +25,7 @@ github_org:
         Set to your GitHub username to make a personal repo, or DiamondLightSource
         if you have permissions to make it in the DiamondLightSource organisation.
     when: "{{ git_platform == 'github.com' }}"
+    default: DiamondLightSource
```

Comment:
Interesting, I did not realise that. That's a bit annoying as it incurs extra work for the admin to rename all references to the org when they transfer. I'd prefer the docs said "ask an admin to make a repo for you unless it's a very speculative project, then make it yourself and ask an admin to transfer it in when you're more certain". Then hopefully `DiamondLightSource` _becomes_ the majority case.

Reply:
It's only an issue for references that are added. If you `copier update` and provide a different answer for `github_org` then it will update all the templated references for you.

Either way, there is no way to default when in backstage (the UI element won't let you default), so I don't really mind either way here, only people adopting from skeleton will see this default, so maybe I agree this should be the default.

We can argue about the backstage docs another time...